articles/search/semantic-search-overview.md

```diff
@@ -115,23 +115,36 @@ By default, semantic search is disabled on all services. To enable semantic sear
 
 Semantic Search's free plan is capped at 1,000 queries per month. After the first 1,000 queries in the free plan, you'll receive an error message letting you know you've exhausted your quota whenever you issue a semantic query. When this happens, you need to upgrade to the standard plan to continue using semantic search.
 
-Alternatively, you can also enable semantic search using the [Create or Update Service API](/rest/api/searchmanagement/2021-04-01-preview/services/create-or-update#searchsemanticsearch) that's described in the next section.
+Alternatively, you can also enable semantic search using the REST API that's described in the next section.
 
-## Disable semantic search
+## Enable semantic search
 
-To reverse feature enablement, or for full protection against accidental usage and charges, you can [disable semantic search](/rest/api/searchmanagement/2021-04-01-preview/services/create-or-update#searchsemanticsearch) using the Create or Update Service API on your search service. After the feature is disabled, any requests that include the semantic query type will be rejected.
+To enable Semantic Search using the REST API, you can use the [Create or Update Service API](/rest/api/searchmanagement/2021-04-01-preview/services/create-or-update#searchsemanticsearch).
+
+> [!NOTE]
+> Create or Update supports two HTTP methods: PUT and PATCH. PUT replaces all properties in the service with their defaults if they are not specified in the request. PATCH only replaces properties that are specified in the request. It's possible to accidentally introduce an unexpected scaling or configuration change if PUT is used. It's recommended to use PATCH when enabling semantic search.
```

Comment:
I might specify the risk of PUT is only limited to existing services. If you're attempting to create a new service with semantic search enabled, you'll still need to use PUT.